src/main/java/uk/gov/digital/ho/hocs/queue/ukvi/UKVIComplaintQueueListener.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.digital.ho.hocs.queue.ukvi;
+
+import org.springframework.cloud.aws.messaging.listener.SqsMessageDeletionPolicy;
+import org.springframework.cloud.aws.messaging.listener.annotation.SqsListener;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UKVIComplaintQueueListener {
```

Comment:
outside of the scope of the PR, but seeing a tenant in the code makes me sad.

Reply:
Sado 🙁